libs/execution_context/ebpf_core.c

```diff
@@ -1750,6 +1756,34 @@ _ebpf_core_get_pid_tgid()
     return ((uint64_t)ebpf_platform_process_id() << 32) | ebpf_platform_thread_id();
 }
 
+static uint64_t
+_ebpf_core_get_current_logon_id(_In_ const void* ctx)
+{
+    uint64_t logon_id = 0;
+
+    UNREFERENCED_PARAMETER(ctx);
+
+    if (!ebpf_is_preemptible()) {
+        return 0;
+    }
+
+    ebpf_result_t result = ebpf_platform_get_authentication_id(&logon_id);
+    if (result != EBPF_SUCCESS) {
+        return 0;
```

Comment:
If this is an instance of a 'benign' and frequent error, a comment to that effect (in lieu of error logging) would be helpful.

Reply:
Added failure traces inside `ebpf_platform_get_authentication_id`